conda-forge.yml

```diff
@@ -13,8 +12,6 @@ idle_timeout_minutes: 60
 os_version:
   linux_64: cos7
   linux_aarch64: cos7
```

Comment:
```suggestion
```
The test time glibc error is because cublasLt needs a later version of glibc than this package. Since we are checking the glibc symbols explicitly, we can drop these os_version modifications and float up to the latest container which will provide a newer glibc for the test environment.

Remember to rerender the feedstock afterwards.

Reply:
The builds still fail and the outputs indicate that glibc version is 2.17.

Reply:
Looking at [the pipeline from the previous release](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=999458&view=logs&j=2ed0550f-e734-5ebf-52c6-1bfc19f5811c&t=9c969b19-a14a-5482-8fb9-78bdf745c3c6) (24.08), they seemed to have been skipping the tests for this build config (`aarch64` and cuda major version 12) as well. We could have them skip here as well?

Reply:
The system glibc versions in the containers seems to be 2.34 now, but the recipe's glibc is still at 2.17. Somehow this doesn't create a problem for the x86 builds, but does for the aarch64 ones.

Reply:
- We can currently require c_stdlib_version to be 2.17, 2.28, 2.34 (the container's are all using glibc2.34)
- The current checks in the bash scripts on RECIPE_GBLIC are not always accurate since `c_stdlib_version` doesn't reflect the actual glibc in use unless the constraint is enforced explicitly via build/test requirements.
- Any glibc!=2.17 will break cutensor dependency, since cutensor's conda package seems to pin glibc to 2.17 at runtime ([I opened a PR to fix that](https://github.com/conda-forge/cutensor-feedstock/pull/59)).

If we don't care about test coverage we can:
- keep `c_stdlib_version` at 2.17 in conda_forge_build.yaml.
- use a check against the actual glibc in use in the environment and retain skip logic to skip tests with 0 exit signal.
- this will mean no coverage for: aarch64 and cuda major version 12 for all components, all configs(?) for custatevec

Reply:
This shouldn't be happening. Binaries in the test environment should be able to pick up glibc from the container. I'm trying to prototype a workaround right now.

Reply:
Sorry, I forgot to push my changes that led to these insights last night.